go.mod

```diff
@@ -4,27 +4,30 @@ module k8s.io/test-infra
 
 replace github.com/golang/lint => golang.org/x/lint v0.0.0-20190301231843-5614ed5bae6f
 
-// Pin all k8s.io staging repositories to kubernetes v0.17.3
+// Pin all k8s.io staging repositories to kubernetes v0.18.6
 // When bumping Kubernetes dependencies, you should update each of these lines
 // to point to the same kubernetes v0.KubernetesMinor.KubernetesPatch version
 // before running update-deps.sh.
 replace (
+	cloud.google.com/go/pubsub => cloud.google.com/go/pubsub v1.3.1
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v12.2.0+incompatible
+	github.com/docker/docker => github.com/docker/docker v1.4.2-0.20200203170920-46ec8731fbce
```

Comment:
Not sure why, but without the replacements, `./hack/update-deps.sh` would update the pubsub and docker dependency, causing compilation failures.

I've pinned all the k8s dependencies to make sure we use the same version for everything